wallet/build.gradle

```diff
@@ -357,6 +367,15 @@ android {
             freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"
         }
     }
+    lint {
+        abortOnError false
+        disable 'MissingTranslation'
+    }
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+        kotlinOptions {
+            freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"
+        }
+    }
```

Comment:
I think this code duplicates the section above

Reply:
You are right, this can be removed.